modules/nf-neuro/registration/anattodwi/main.nf

```diff
@@ -32,34 +36,41 @@ process REGISTRATION_ANATTODWI {
     export ANTS_RANDOM_SEED=1234
 
     antsRegistration --dimensionality 3 --float 0\
-        --output [output,outputWarped.nii.gz,outputInverseWarped.nii.gz]\
+        --output [trans,warped.nii.gz]\
         --interpolation Linear --use-histogram-matching 0\
         --winsorize-image-intensities [0.005,0.995]\
-        --initial-moving-transform [$b0,$t1,1]\
+        --initial-moving-transform [$fixed_reference,$moving_anat,1]\
         --transform Rigid['0.2']\
-        --metric MI[$b0,$t1,1,32,Regular,0.25]\
+        --metric MI[$fixed_reference,$moving_anat,1,32,Regular,0.25]\
         --convergence [500x250x125x50,1e-6,10] --shrink-factors 8x4x2x1\
         --smoothing-sigmas 3x2x1x0\
         --transform Affine['0.2']\
-        --metric MI[$b0,$t1,1,32,Regular,0.25]\
+        --metric MI[$fixed_reference,$moving_anat,1,32,Regular,0.25]\
         --convergence [500x250x125x50,1e-6,10] --shrink-factors 8x4x2x1\
         --smoothing-sigmas 3x2x1x0\
         --transform SyN[0.1,3,0]\
-        --metric MI[$b0,$t1,1,32]\
-        --metric CC[$metric,$t1,1,4]\
+        --metric MI[$fixed_reference,$moving_anat,1,32]\
+        --metric CC[$metric,$moving_anat,1,4]\
         --convergence [50x25x10,1e-6,10] --shrink-factors 4x2x1\
         --smoothing-sigmas 3x2x1
```

Comment:
The specific set of shrinks and smooths here is likely stable only for human-shaped and isotropic data.

Reply:
Yes. We should rename and update the documentation to reflect that. It's only doing the work of Tractoflow, for acquisitions on healthy human. I hope one day this module can generalize, but for now it serves the only purpose of being **the reference T1 -> DWI** registration workflow.